test/unit/ImportExport/MDL/utMDLImporter_HL1_Nodes.cpp

```diff
@@ -55,6 +55,17 @@ using namespace Assimp;
 
```

Comment:
This is a really great unittest to validate the imported data. Thank you so much.